template/{{cookiecutter.module_name}}/1.0/{{cookiecutter.module_name}}.smk

```diff
@@ -135,7 +157,7 @@ rule _{{cookiecutter.module_name}}_step_1:
     threads:
         CFG["threads"]["step_1"]
     resources:
-        mem_mb = CFG["mem_mb"]["step_1"]
+        **CFG["resources"]["step_1"]
```

Comment:
Can you please also add commented documentation here to explain the resource unpacking?

Reply:
Done